modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -936,7 +934,8 @@ private void processOperation(String resourcePath, String httpMethod, Operation
                 for (String tagName : tagNames) {
                     boolean foundTag = false;
                     for (Tag tag : swaggerTags) {
-                        if (tag.getName().equals(tagName)) {
+                        String name = tag.getName();
+                        if (null != name && name.equals(tagName)) {
```

Comment:
Minor suggestion: use `name != null` instead to make it more consistent with the rest of the code base.

Reply:
I prefer starting with the value to prevent accidental ‘name = null’ typos which are hard to debug, but if you prefer I’ll switch them around for consistency.